content/docs/user-guide/data-management/cloud-versioning.md

```diff
@@ -0,0 +1,113 @@
+# Cloud Versioning
+
+<admon type="warn">
+
+Cloud versioning features are currently under active development and should be
+considered experimental. These features are subject to frequent change, and the
+documentation may not always reflect changes available in the latest DVC
+release.
+
+</admon>
+
+When cloud versioning is enabled, DVC will store files in the remote according
+to their original directory location and filenames. Different versions of a file
+will then be stored as separate versions of the corresponding object in cloud
+storage. This is useful for cases where users prefer to retain their original
+filenames and directory hierarchy in remote storage (instead of using DVC's
+usual
+[content-addressable storage](/doc/user-guide/project-structure/internal-files#structure-of-the-cache-directory)
+format).
+
+<admon type="warn">
+
+Note that not all DVC functionality is supported when using cloud versioned
+remotes, and using cloud versioning comes with the tradeoff of losing certain
+benefits of content-addressable storage.
```

Comment:
What specific functionality is not available? The hidden section below only states that "DVC does not provide de-duplication, and certain remote storage performance optimizations will be unavailable".

Reply:
pretty much anything that isn't explicitly documented as a cloud-versioning-enabled command/feature

But off the top of my head, some of the things that don't work are:

- `dvc import` from a DVC repo that uses cloud versioned remote(s)
- run cache (it works locally but you cannot push/pull it with cloud versioned remotes)
- `gc -c`
- `status -c`
- `exp` sharing (anything that requires `exp push/pull`)
- `dvc push` with revision flags (you cannot `dvc push --all-branches/--all-tags/--all-commits` to cloud versioned remotes, but `dvc pull` will work with the flags)

Reply:
It may be worth it to rephrase this and other texts then, to emphasize that A LOT of features don't work with cloud versioned data remotes.